tests/features/steps/steps.py

```diff
@@ -0,0 +1,43 @@
+import os
+import requests
+from behave import given, when, then
+
+from utils import table_to_json
+
+
+@given('I have the API endpoint "{endpoint}"')
+def step_given_api_endpoint(context, endpoint):
```

Comment:
Think this step is useless, because you have a before or constructor to initialise context, check leto-modelizer

Reply:
done